universal-application-tool-0.0.1/app/auth/UatProfileData.java

```diff
@@ -18,19 +16,10 @@
  * <p>It is wrapped by UatProfile, which is what we should use server-side.
  */
 public class UatProfileData extends CommonProfile {
-  private Clock clock;
 
-  public UatProfileData() {
-    this(Clock.system(ZoneId.of("America/Los_Angeles")));
-  }
-
-  public UatProfileData(Clock clock) {
-    super();
-    this.clock = Preconditions.checkNotNull(clock);
-  }
+  public UatProfileData() {}
```

Comment:
suggestion (blocking): I think we still want to call `super()` here, so that the parent class constructor is called